EasyPeasy.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "EasyPeasy"
-  s.version          = "1.5.0"
+  s.version          = "1.5.1"
```

Comment:
In this time, We have big changes deprecating operator (this was the key feature).
So, I think we need to update to `1.6.0` or `2.0.0`.
What do you think?

Reply:
Yup, I'm going to release `1.6.0` with the old methods marked as deprecated and then after sometime will release `2.0.0` forbidding the use of the deprecated ones.

Reply:
@nakiostudio 
Okay!